src/libstd/io/mod.rs

```diff
@@ -1458,7 +1458,7 @@ pub trait BufRead: Read {
     ///
     /// If successful, this function will return the total number of bytes read.
     ///
-    /// An empty buffer returned indicates that the stream has reached EOF.
```

Comment:
For clarity, can we say "`Ok(0)` returned"?

Reply:
How about:

```markdown
If this function returns `Ok(0)`, the stream has reached EOF.
```

Reply:
Sounds great, :shipit:.